src/h3lib/lib/algos.c

```diff
@@ -205,26 +203,27 @@ H3Index h3NeighborRotations(H3Index origin, const CoordIJK* translationIjk,
     int dir = _unitIjkToDigit(&adjustment);
 
     int newRotations = 0;
-    int oldBaseCell = hf.baseCell;
-    int oldLeadingDigit = _leadingNonZeroDigit(&hf);
```

Comment:
i assume these functions will be removed when you follow up with the `H3IndexFat` removal?